dist/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>API</title>
+<script defer src="main.js"></script></head>
+<body id="body">
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">Logo</a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarCollapse">
+            <ul class="navbar-nav me-auto mb-2 mb-md-0">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Movies</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Bookmarked</a>
+              </li>
+            </ul>
+            <form class="d-flex">
+              <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+              <button class="btn btn-outline-success" type="button">Search</button>
+            </form>
+          </div>
+        </div>
+      </nav>
+      <div class="alert alert-secondary text-center" role="alert">
+        A simple secondary alert—check it out!
+      </div>
+      <section class="items-list container-md my-5">
+        <div class="card d-none">
+          <img src="#" class="card-img-top img-1" alt="...">
+          <div class="card-body">
+            <h5 class="card-title">Card title</h5>
+            <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+            
+            <a href="#">...see more</a>
+          </div>
+          <div class="card-footer">
+            <div class="d-flex justify-content-between">   
+              <a class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal"  data-action="comment" id="commentButton">Comment</a>
+              <button type="button" class="card-body__likebtn">👍🏼</button>
```

Comment:
- [ ] After liking an item ensure that the likes count updates automatically.

Reply:
Hello Mike, 
- This is a video to illustrate that the ![Like Button is working](https://drive.google.com/file/d/1Rly_kbHND2sUS_RKIKu0RhiWHhj6T4tJ/view?usp=sharing)